tests/testthat/test-rmnorm.R

```diff
@@ -0,0 +1,30 @@
+test_that("Check input", {
```

Comment:
I'm not really sure that tests for `rmnorm()` are necessary 😩 
I don't think we use `rmnorm()` in any of our functions as we only used it for the naive implementation of ltfh assignment. Am I wrong in this?

My point is that if we do not use it the package, we might want to remove it from the package completely🤔

Reply:
My understanding is that most functions should be tested. I do not know if the function is used or not, I just remeber that Emil wanted us to implement it (I think so we understand how it worked). Regardless I think it is fine, that the function is in the package even if the user might never see or use it ;).